client/src/components/Users.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+const UserList = styled.div`
+  position: relative;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+`;
+
+class Users extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      users: [],
+      errorMessage: '',
+    }
+  }
+
+  componentDidMount() {
+    axios.defaults.withCredentials = true;
```

Comment:
That's a pretty clean place to put this line. I think I prefer this way to all the other ways I've seen.